templates/unitarity.cpp.in

```diff
@@ -0,0 +1,76 @@
+// ====================================================================
+// This file is part of FlexibleSUSY.
+//
+// FlexibleSUSY is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published
+// by the Free Software Foundation, either version 3 of the License,
+// or (at your option) any later version.
+//
+// FlexibleSUSY is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with FlexibleSUSY.  If not, see
+// <http://www.gnu.org/licenses/>.
+// ====================================================================
+
+
+/**
+ * @file @ModelName@_unitarity.cpp
+ *
+ * This file was generated with FlexibleSUSY @FlexibleSUSYVersion@ and SARAH @SARAHVersion@ .
+ */
+
+#include "@ModelName@_unitarity.hpp"
+#include "@ModelName@_mass_eigenstates.hpp"
+#include "cxx_qft/@ModelName@_fields.hpp"
+
+#include "sum.hpp"
+#include "wrappers.hpp"
+
+#include <Eigen/Eigenvalues>
+
+namespace flexiblesusy {
+namespace @ModelName@_unitarity {
+
+static constexpr int size = @scatteringPairsLength@;
+
+inline
```

Comment:
I'd recommend to put these three `inline` functions into an anonymous namespace, which expresses the fact that these functions are local to this translation unit.

Reply:
Fixed in 366bca7cb9d71186a6c903a32f53318c997e2a56.